xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -1174,10 +1174,11 @@
       "fulltextepgsearch": { "type": "boolean" },
       "recordinggroup": { "type": "integer" },
       "maxrecordings": { "type": "integer" },
-      "epguid": { "type": "integer" },
+      "epguid": { "type": "integer", "description": "Deprecated - use broadcastid instead" },
```

Comment:
this comment is misleading. There is still epguid and for other use cases, not the one described in the issue #18960, it might still be useful/needed.

Reply:
Ok, I’ll remove the this change. I’m not sure what use cases these would be. I don’t think there is any value or meaning for this variable in the JSOn api.

Reply:
Done

Reply:
Really? Can't see the change.

Reply:
Oops, push old branch. It's there now.